spec/lib/percy/capybara_spec.rb

```diff
@@ -76,4 +76,14 @@
       expect(Percy::Capybara.capybara_client.enabled?).to eq(false)
     end
   end
+  describe '#use_loader' do
+    class DummyLoader < Percy::Capybara::Loaders::NativeLoader; end
+
+    it 'sets the current capybara clients custom loader' do
+      capybara_client = Percy::Capybara::Client
+      expect(Percy::Capybara.capybara_client.custom_loader).to be nil
```

Comment:
I don't usually pass anything into `be`, just use it as a check for existence or not. The normal form is to to use `.to_not be` or the more explicit `.to be_nil` here, and then `.to be` or the more explicit `.to_not be_nil` below, but I prefer the shorter form.